go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/viper v1.15.0
 	github.com/t-tomalak/logrus-easy-formatter v0.0.0-20190827215021-c074f06c5816
-	github.com/yugabyte/yugabytedb-managed-go-client-internal v0.0.0-20230418071958-56cbcbea1ba8
+	github.com/yugabyte/yugabytedb-managed-go-client-internal v0.0.0-20230511090901-1971858d1c13
```

Comment:
@arishta-yb - this change is only required when we bump the managed client version, right?

Reply:
Yes, but the string to enum change for CMKProviderEnum was in the updated version of the managed client.